include/snowflake/Param.hpp

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2017-2018 Snowflake Computing, Inc. All rights reserved.
+ */
+
+#ifndef SNOWFLAKECLIENT_PARAM_HPP
+#define SNOWFLAKECLIENT_PARAM_HPP
+
+#include "client.h"
+
+namespace Snowflake {
+    namespace Client {
+        class Param {
+        public:
+            Param();
+
+            ~Param();
+
+            size_t len();
+
+            void set_len(size_t len_);
+
+            SF_DB_TYPE dbDataType();
+
+            void set_dbDataType(SF_DB_TYPE type_);
```

Comment:
setDbDataType?

Reply:
I can change all of the set functions that format. At school the convention is: 

```
// set function
void set_<variable_name>(int value)

// get function
int <variable_name>()
```